src/main/scala/org/globalforestwatch/summarystats/annualupdate_minimal/AnnualUpdateMinimalSummary.scala

```diff
@@ -109,15 +109,28 @@ object AnnualUpdateMinimalSummary {
         val area: Double = Geodesy.pixelArea(lat, raster.cellSize) // uses Pixel's center coordiate.  +- raster.cellSize.height/2 doesn't make much of a difference
 
         val areaHa = area / 10000.0
-
         val gainArea: Double = gain * areaHa
+        val biomassPixel = biomass * areaHa
 
         val co2Factor = 0.5 * 44 / 12
-
-        val biomassPixel = biomass * areaHa
         val co2Pixel = biomassPixel * co2Factor
-        //        val mangroveBiomassPixel = mangroveBiomass * areaHa
-        //        val mangroveCo2Pixel = mangroveBiomassPixel * co2Factor
+
+        val grossEmissionsCo2eNonCo2: Float = raster.tile.grossEmissionsCo2eNonCo2.getData(col, row)
+        val grossEmissionsCo2eCo2Only: Float =  raster.tile.grossEmissionsCo2eCo2Only.getData(col, row)
+        val grossCumulAbovegroundRemovalsCo2: Float =
+          raster.tile.grossCumulAbovegroundRemovalsCo2.getData(col, row)
+        val grossCumulBelowgroundRemovalsCo2: Float =
+          raster.tile.grossCumulAbovegroundRemovalsCo2.getData(col, row)
```

Comment:
grossCumulBelowgroundRemovalsCo2 is using grossCumulAbovegroundRemovalsCo2. Correct to grossCumulBelowgroundRemovalsCo2. This is making the belowground removals have the same values as the aboveground removals.

Reply:
good catch!